setuptools_rust/setuptools_ext.py

```diff
@@ -152,19 +152,10 @@ def run(self) -> None:
 
     install_base_class = cast(Type[install], dist.cmdclass.get("install", install))
 
-    # this is required because, install directly access distribution's
-    # ext_modules attr to check if dist has ext modules
+    # this is required for console_scripts and to make install_scripts compatible with RustBin
     class install_rust_extension(install_base_class):  # type: ignore[misc,valid-type]
         def finalize_options(self) -> None:
-            ext_modules = self.distribution.ext_modules
-
-            # all ext modules
-            mods = []
-            if self.distribution.ext_modules:
-                mods.extend(self.distribution.ext_modules)
             if self.distribution.rust_extensions:
-                mods.extend(self.distribution.rust_extensions)
-
                 scripts = []
                 for ext in self.distribution.rust_extensions:
                     scripts.extend(ext.entry_points())
```

Comment:
I wonder if we can get rid of this.

Reply:
Maybe needed by the [scripts/exec bindings](https://github.com/PyO3/setuptools-rust/blob/v1.4.1/setuptools_rust/extension.py#L216-L224) still?

Reply:
Yes, I'm thinking about whether it's possible to move it to `self.distribution` so that we don't need to override `finalize_options`.

Reply:
Probably a good idea to have a regular binding.exec+script style test similar to [the hello world one](https://github.com/PyO3/setuptools-rust/blob/main/examples/hello-world/setup.py).